Samples/Cordova/ExampleProject/platforms/android/CordovaLib/src/org/apache/cordova/PluginManager.java

```diff
@@ -617,4 +619,29 @@ public ArrayList<CordovaPluginPathHandler> getPluginPathHandlers() {
         }
         return handlers;
     }
+
+    /**
+     * Called when the WebView's render process has exited.
+     *
+     * See https://developer.android.com/reference/android/webkit/WebViewClient#onRenderProcessGone(android.webkit.WebView,%20android.webkit.RenderProcessGoneDetail)
+     *
+     * @return  true if the host application handled the situation that process has exited,
+     *          otherwise, application will crash if render process crashed, or be killed 
+     *          if render process was killed by the system.
+     */
+    public boolean onRenderProcessGone(final WebView view, RenderProcessGoneDetail detail) {
+        boolean result = false;
+        synchronized (this.entryMap) {
+            for (PluginEntry entry : this.entryMap.values()) {
+                CordovaPlugin plugin = pluginMap.get(entry.service);
+                if (plugin != null) {
+                    if (plugin.onRenderProcessGone(view, detail)) {
+                        result = true;
+                    }
+                }
+            }
```

Comment:
`synchronized ` might not be needed as you are already using a syncronised map.

Reply:
These are all auto generated files by cordova